app/views/layouts/_sorting_bar.haml

```diff
@@ -0,0 +1,34 @@
+%nav.navbar.bg-light.sorting-nav
+  .mr-auto
+    .dropdown
+      %button.dropbtn
+        Category
+        %i.fa.fa-angle-down
+      / .dropdown-content
+      /   - @categories = @categories.nil? ? Category.all : @categories
+      /   - @categories.each do |category|
+      /     %a{:href => category_path(category)}= category.title
+    .dropdown
+      %button.dropbtn
+        Sorting
+        %i.fa.fa-angle-down
+      .dropdown-content
+        %a{ :href => '#' } Link 1
+    .dropdown
+      %button.dropbtn
+        Vegetarian
+        %i.fa.fa-angle-down
+      .dropdown-content
+        %a{ :href => '#' } Link 1
+  .ml-auto{ :style => 'align-content: center;' }
+    .score
+      Price:
+    #slider-value-1.score
+      $0
+    .score
+      .slidecontainer
+        .range-slider
+          %input{ :max => '100', :min => '1', :step => '1', :type => 'range', :value => '1' }/
```

Comment:
why do you use hardcoded values? Max 100, but what if you have products which costs 200?

Reply:
Code was not modified since the first assignment, and was kept to maintain design somewhat. Will adjust accordingly if changes are needed.

Reply:
ok